python/ray/autoscaler/_private/aws/config.py

```diff
@@ -337,11 +337,7 @@ def _configure_key_pair(config):
             "No matching local key file for any of the key pairs in this "
             "account with ids from 0..{}. "
             "Consider deleting some unused keys pairs from your account.",
-            key_name)  # todo: err msg
-        raise ValueError(
-            "No matching local key file for any of the key pairs in this "
-            "account with ids from 0..{}. ".format(key_name) +
-            "Consider deleting some unused keys pairs from your account.")
+            key_name)
```

Comment:
ah I think the reason why we had these was because you could somehow disable cli_logger.abort.

Let me check if that this is for sure dead code.

Reply:
ok it looks fine. let's be sure to keep an eye out in case someone starts reporting weird issues.

Reply:
yeah I suspected that as well considering how widespread the pattern was, or maybe cli_logger.abort used to behave like logging.exception and did not actually raise the exception. Anyway, the abort() definition today seems to always raise an exception now.